deploy/helm/lunettes/templates/elasticsearch/elasticsearch-clean-job.yaml

```diff
@@ -1,4 +1,8 @@
+{{- if semverCompare ">=1.21-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
Why not use the same method here as in the previous policyversion?

Reply:
> Why not use the same method here as in the previous policyversion?

Directly using nested yaml files will result in an error and requires passing variables